docs/mine-hnt/miner-cli/minercli.mdx

```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
----
+# Introduction
```

Comment:
I think you need a line break below (and maybe above) this for markdown to pick it up.

Reply:
OK, weirdness is when in my local Yarn it works but the online preview doesn't work. 

Fixed it.